xbmc/cores/VideoPlayer/VideoRenderers/DebugRenderer.h

```diff
@@ -32,7 +39,7 @@ class CDebugRenderer
     void Render(int idx) override;
   };
 
-  std::string m_strDebug[4];
-  CDVDOverlayText *m_overlay[4];
+  std::string m_strDebug[MAX_LINES];
```

Comment:
I think `std::array<std::string, MAX_LINES>` would be better or even better would be to use `std::vector<std::string>` as we don't have a defined size at compile time. This would also simplify some of the interface methods